<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ACGTagGenerator</class>
 <widget class="QWidget" name="ACGTagGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ACGTagGenerator</string>
  </property>
  <widget class="QTreeWidget" name="treeWidgetWork">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>262</width>
     <height>481</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>262</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>262</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <column>
    <property name="text">
     <string>作品角色</string>
    </property>
   </column>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>580</y>
     <width>531</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>511</width>
      <height>104</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>場次號</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox">
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="value">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Generate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonClear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>801</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTreeWidget" name="treeWidgetCommon">
      <property name="tabKeyNavigation">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <column>
       <property name="text">
        <string>一般</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QTreeWidget" name="treeWidgetActivity">
      <property name="tabKeyNavigation">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>活動場次</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QTreeWidget" name="treeWidgetLocation">
      <property name="tabKeyNavigation">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>地點</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>treeWidgetWork</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
